webrtc.html

```diff
@@ -8624,14 +8652,24 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                 <li>Set the object's <code><a data-for=
                 "RTCDTMFSender">toneBuffer</a></code> attribute to
                 <var>tones</var>.</li>
+                <li>Set <var>dtmf</var>'s <a>[[\Duration]]</a> slot to the
+                value of the <code><a data-for=
+                "RTCDTMFSender">duration</a></code> parameter.</li>
```

Comment:
These links don't seem to work. But I also don't get a respec error, as I'd expect. Any idea what's going on?